releasenotes/notes/bugfix_expdata_copy-2c73a21ad720858d.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    Added a warning if `ExperimentData` object data's is empty and there is no provider to retrieve data
+    from.
```

Comment:
```suggestion
    During data retrieval, added a warning if `ExperimentData` object data is empty and there is no provider to retrieve data
    from.
```

Reply:
Removed this part from the release notes as discussed here.